inst/Upgrades.qml

```diff
@@ -58,4 +58,858 @@ Upgrades
 			}
 		}
 	}
+
+	Upgrade
+	{
+		functionName: "LatentGrowthCurve";	fromVersion: "0.15";	toVersion: "0.16";
+
+		ChangeRename
+		{
+			from:	"covar"
+			to:		"covaryingLatentCurve"
+		}
+
+		ChangeRename
+		{
+			from:	"outputAdditionalFitMeasures"
+			to:		"additionalFitMeasures"
+		}
+
+		ChangeRename
+		{
+			from:	"rsquared"
+			to:		"rSquared"
+		}
+
+		ChangeRename
+		{
+			from:	"std"
+			to:		"standardizedEstimate"
+		}
+
+		ChangeRename
+		{
+			from:	"impliedCov"
+			to:		"impliedCovarianceMatrix"
+		}
+
+		ChangeRename
+		{
+			from:	"residCov"
+			to:		"residualCovarianceMatrix"
+		}
+
+		ChangeRename
+		{
+			from:	"showSyntax"
+			to:		"syntax"
+		}
+
+		ChangeRename
+		{
+			from:	"curveplot"
+			to:		"curvePlot"
+		}
+
+		ChangeRename
+		{
+			from:	"plot_categorical"
+			to:		"curvePlotCategorical"
+		}
+
+		ChangeRename
+		{
+			from:	"plot_max_n"
+			to:		"curvePlotMaxLines"
+		}
+
+		ChangeRename
+		{
+			from:	"misfitplot"
+			to:		"misfitPlot"
+		}
+
+		ChangeRename
+		{
+			from:	"pathplot"
+			to:		"pathPlot"
+		}
+
+		ChangeRename
+		{
+			from:	"plotpars"
+			to:		"pathPlotParameter"
+		}
+
+		ChangeRename
+		{
+			from:	"plotmeans"
+			to:		"pathPlotMean"
+		}
+
+		ChangeRename
+		{
+			from:	"mimic"
+			to:		"emulation"
+		}
+
+		ChangeJS
+		{
+			name:		"emulation"
+			jsFunction:	function(options)
+			{
+				switch(options["emulation"])
+				{
+					case "Mplus":	return "mplus";
+					case "EQS":	return "eqs";
```

Comment:
if there are RadioButtons that were not changed like here (i.e., `lavaan`), you'll need to catch them with a default statement
```suggestion
					case "EQS":	return "eqs";
					default:	return options["emulation"];
```